app/Controller/PagesController.php

```diff
@@ -74,3 +115,44 @@ public function display() {
 		}
 	}
 }
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
```

Comment:
[code_sniffer] Opening brace of a class must be on the line after the definition